iceoryx_posh/test/moduletests/test_roudi_service_registry.cpp

```diff
@@ -586,15 +591,15 @@ string_t randomString(uint64_t size = string_t::capacity())
     constexpr auto N = string_t::capacity();
     size = std::min(N, size);
 
-    char a[N + 1];
+    char a[N + 1U];
```

Comment:
:heart:

Reply:
These Us have to go actually. Seriously, its nothing but clutter. And in many cases (`uint64_t`) the actual literal type is `ULL` and we have a "conversion" (as we would have from a signed literal). Now, the `signedness matches`, but the compiler does the correct thing in these cases anyway (only problem is when the literals get large).